src/models/clinics.models.ts

```diff
@@ -0,0 +1,31 @@
+import db from '../database/connection';
+
+export function fetchProtocols() {
+  return db.query('SELECT * FROM protocols').then((protocols) => {
+    return protocols.rows;
+  });
+}
+
+export function fetchSurveys() {
+  return db.query('SELECT * FROM surveys').then((surveys) => {
+    return surveys.rows;
+  });
+}

```

Comment:
to count rows in a different table check this https://dba.stackexchange.com/questions/110850/count-rows-with-inner-joined-tables

essentially what you need to is say `LEFT JOIN` and then a sub query which returns a table of for example the counts of surveys and the `survey_id`

`LEFT JOIN (SELECT survey_id, COUNT(survey_id))`, and I'll keep the rest secret 😄 . the hint is `GROUP BY`